AppJiggling/AppJiggling/HDock.swift

```diff
@@ -45,7 +45,7 @@ struct HDock: View {
                 
                 LazyVGrid(columns: rows, spacing: apps.count > 4 ? 19 : 0) {
                     
-                    ForEach(apps.indices, id: \.self) { index in
+                    ForEach(Array(zip(apps.indices, apps)), id: \.1) { (index, app) in
                         
                         Apps(
                             apps: $apps,
```

Comment:
With this change, couldwe just do `appNames: app` ?

Reply:
It would instead be `appNames: [app]`, but yes. I tried to keep the PR changes minimal for simplicity on purpose, so I didn't change this. That is a change I would indeed make though.